### PR TITLE
add some export-importer tests

### PR DESCRIPTION
**Pull request by mikehelmick**
## Proposed Changes

* adds tests. Tests are good.

**Release Note**

```release-note
NONE
```

### CODE REVIEW

File: internal/exportimport/scheduler.go [hunk 1]
```diff
@@ -116,3 +106,27 @@ func (s *Server) handleSchedule(ctx context.Context) http.HandlerFunc {
...
+
+func syncFilesFromIndex(ctx context.Context, db *exportimportdb.ExportImportDB, config *model.ExportImport, index string) (int, error) {
+	zipNames := strings.Split(index, "\n")
+	currentFiles := make([]string, 0, len(zipNames))
+	for _, zipFile := range zipNames {
+		if len(strings.TrimSpace(zipFile)) == 0 {
+			// drop blank lines.
+			continue
+		}
+		proposedPath := fmt.Sprintf("%s%s", config.ExportRoot, strings.TrimSpace(zipFile))
```
Comment (jeremyfaller) [inline internal/exportimport/scheduler.go:118]: path.Join?

Comment (sethvargo) [reply internal/exportimport/scheduler.go:118]: `filepath.Join` in case someone runs this on Windows ever

Comment (mikehelmick) [reply internal/exportimport/scheduler.go:118]: these aren't file paths, they're URL paths

path.Join doesn't do well when the protocol/hostname is in play

You end up with `https:///myserver/a.zip` if Join is used.

Comment (jeremyfaller) [reply internal/exportimport/scheduler.go:118]: perhaps rename paths to urls?

Comment (jeremyfaller) [reply internal/exportimport/scheduler.go:118]: perhaps rename paths to urls?

Comment (mikehelmick) [reply internal/exportimport/scheduler.go]: done

**Review by sethvargo — APPROVED** [review]
/lgtm

**Pull request merged**